packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -364,6 +365,14 @@ export default class QuanticNumericFacet extends LightningElement {
     });
   }
 
+  onChangeMin(evt) {
```

Comment:
you add those event to html?

Reply:
i had but the commit didnt pick it up 🤦

Reply:
I love the title: `fix(quantic): harry potter and the lost changeset` 😆